src/context/TokenContext.js

```diff
@@ -1,5 +1,54 @@
-import React from "react";
+import React, { createContext, useContext, useEffect, useState } from "react";
+import jwtDecode from "jwt-decode";
+
+const initialState = { session: null, user: null, username: null };
```

Comment:
init fields would be `{token:null, authenticated:false}`

Reply:
✅